### PR TITLE
Device remove fix

### PR DESCRIPTION
**Pull request by riomus**
Using object path in device remove. Before fix, string was used what generated exception during object removal.

### CODE REVIEW

**Comment by dati91** [comment]
Hi!
You are right! Thanks for noticing and making this pr.

File: src/bluetooth_adapter.rs [hunk 1]
```diff
@@ -227,6 +227,8 @@ impl BluetoothAdapter {
 
     // http://git.kernel.org/cgit/bluetooth/bluez.git/tree/doc/adapter-api.txt#n40
     pub fn remove_device(&self, device: String) -> Result<(), Box<Error>> {
-        self.call_method("RemoveDevice", Some([device.into()]))
+        let path:Path=device.into();
+        let message_item=MessageItem::ObjectPath(path);
+        self.call_method("RemoveDevice", Some([message_item]))
```
Comment (dati91) [inline src/bluetooth_adapter.rs]: Can you shorten this? something like
`self.call_method("RemoveDevice", Some([MessageItem::ObjectPath(device.into())]))`
should work. And then the `Path` include shouldn't be necessary either.

File: examples/test3.rs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+use std::error::Error;
+use std::time::Duration;
+use std::thread;
+
+use blurz::bluetooth_adapter::BluetoothAdapter as Adapter;
+use blurz::bluetooth_device::BluetoothDevice as Device;
+use blurz::bluetooth_discovery_session::BluetoothDiscoverySession as DiscoverySession;
+
+fn test2() -> Result<(), Box<Error>> {
```
Comment (dati91) [inline examples/test3.rs]: `test2` -> `test3`

File: examples/test3.rs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    let adapter: Adapter = try!(Adapter::init());
+    let session = try!(DiscoverySession::create_session(adapter.get_id()));
+    loop{
```
Comment (dati91) [inline examples/test3.rs]: `loop{` -> `loop {`

**Comment by riomus** [comment]
Fixed things from your comments

**Comment by dati91** [comment]
Great!

**Pull request merged**